kubernetes/deployment/controller.go

```diff
@@ -0,0 +1,282 @@
+// Copyright © 2018 Genome Research Limited
+//  Author: Theo Barber-Bany
+// <tb15@sanger.ac.uk>.
```

Comment:
Sorry to be a pain over the trivial, but your email address goes on the same line as Author, which shouldn't have an extra space before it. Hopefully you can easily fix all files at once with a bit of regex magic...

Reply:
Ah for Pete's sake, it's the rewrap on save. And I'm not too sure how to do all at once (bash?)